fe2-android/app/src/test/java/com/github/dedis/popstellar/ui/detail/LaoDetailActivityTest.java

```diff
@@ -146,6 +149,13 @@ public void witnessButtonShowsWitnessTab() {
     fragmentContainer().check(matches(withChild(withId(witnessFragmentId()))));
   }
 
+  @Test
+  public void OpenQRSCanningSetsValueCorrectly(){
+    HomeViewModel.HOMEVIEWACTION scan = HomeViewModel.HOMEVIEWACTION.SCAN;
+    HomeViewModel.HOMEVIEWACTION cameraPermission = HomeViewModel.HOMEVIEWACTION.REQUEST_CAMERA_PERMISSION;
+    //TODO: create an instance of HomeViewModel and check if event is well set
+  }
+
```

Comment:
I'd avoid having empty tests except if you plan on filling them in a very short time.
But even then, I think it is best to just push this test once it is complete.

It will avoid issues where dead code is left in the codebase and forgotten.

Reply:
Yeah, I agree forgot that I started writing the test should definitely finish it, but since I don't have enough coverage in the new code there is no fear of merging something unfinished in this case. But yeah should be avoided in the upcoming PRs, thanks